front/src/app/scenes/block/block.component.html

```diff
@@ -23,6 +23,10 @@
           <dd>{{block.gas_limit | bigNumber}}</dd>
           <dt>Gas Used</dt>
           <dd>{{block.gas_used | bigNumber}}</dd>
+          <ng-container *ngIf="block.gas_fees">
+            <dt>Gas Fees</dt>
+            <dd>{{block.gas_fees | weiToGO | bigNumber}}</dd>
+          </ng-container>
```

Comment:
If we pull chain config info to the front end, then we could annotate `Gas Fees` as being transferred to the signer address or burned depending on whether the fork is applied.
![Screenshot from 2021-01-07 09-09-46](https://user-images.githubusercontent.com/1194128/103908525-2aff7c80-50c8-11eb-96e7-afdc1bab5dee.png)
![Screenshot from 2021-01-07 09-10-09](https://user-images.githubusercontent.com/1194128/103908532-2cc94000-50c8-11eb-8a0d-63faba5f91f3.png)